server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -445,6 +446,7 @@ export function create(
 				connectionTrace,
 				readDocumentRetryDelay,
 				readDocumentMaxRetries,
+				redisCacheForGetSession,
```

Comment:
I see we're adding this logic to the GetSession path, but not the CreateDocument path. CreateDocument path is where the initial Session is created. I think we should write to the cache there, as well.

In the case of EC's, this would drop DB access for Session info by 100%. In other cases, it will reduce by at least -1, more in the case where a doc is created, then many clients join simultaneously, in which case the cache wouldn't be written in time to help with the logic as-is.